dracut-module-setup.sh

```diff
@@ -564,13 +608,34 @@ kdump_install_resolv_conf() {
     #
     # [1] https://bugzilla.gnome.org/show_bug.cgi?id=690404
     # [2] https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_networking/manually-configuring-the-etc-resolv-conf-file_configuring-and-managing-networking
-    systemctl -q is-enabled systemd-resolved 2> /dev/null && return 0
     inst "$_resolv_conf"
     if NetworkManager --print-config | grep -qs "^dns=none"; then
         printf "[main]\ndns=none\n" > "${initdir}/${_nm_conf_dir}"/90-dns-none.conf
     fi
 }
 
+kdump_install_ovs_deps() {
+    [[ $has_ovs_bridge == yes ]] || return 0
+    inst_multiple -o $(rpm -ql NetworkManager-ovs) $(rpm -ql openvswitch) /usr/bin/uuidgen /usr/bin/hostname /usr/bin/touch /usr/bin/expr /usr/bin/id /usr/bin/install /usr/bin/setpriv /usr/bin/nice /usr/bin/df
+    # By default, ovsdb-server.service runs as USRE=openvswitch, However
+    # openvswitch doesn't have the permission to write to /tmp in kdump initrd
+    # and ovsdb-service.servie will failure with the error
+    # "ovs-ctl[1190]: ovsdb-server: failed to create temporary file (Permission denied)"
+    # So run ovsdb-server.service as root instead
+    mkdir -p "${initdir}/etc/sysconfig/system.conf.d"
+    echo "root:root" >"${initdir}/etc/sysconfig/openvswitch"
```

Comment:
Shouldn't this be `OVS_USER_ID="root:root"`?

One other option to running OVS as root could be to set TMPDIR to a writable directory, if one exists.

Reply:
Yes, `OVS_USER_ID="root:root"` is what I meant. But your comment made me realize my understanding is incorrect because `echo "root:root" >"${initdir}/etc/sysconfig/openvswitch"` also works. So actually ovsdb-server.service by default run as root unless /etc/sysconfig/openvswitc intructs ovsdb-server.service to run as openvswitch.